packages/analytics/metric-cards/src/utilities/trend-display.ts

```diff
@@ -0,0 +1,51 @@
+export const DECIMAL_DISPLAY = 2
+export const DECIMAL_ROUNDING_PRECISION = DECIMAL_DISPLAY + 1
```

Comment:
Sorry, I mis-typed in my message to you.  This should be +2; I also think it would help to explain that it's for percentages.